.github/workflows/release_nightly.yml

```diff
@@ -304,7 +304,7 @@ jobs:
 
       - name: Publish npm package
         if: ${{ !matrix.demo }}
-        run: npm publish
+        run: npm publish --access public
```

Comment:
Ideally we'd want to continue even if this step fails. This can be accomplished with [`continue-on-error: true`](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error).
Also, it might be worth to add a comment explaining why `--access public` is needed (similar to the description of this PR).